browser/pages/confirm/confirm.html

```diff
@@ -355,6 +355,16 @@
       <span>Red Hat Container Development Kit requires Oracle VirtualBox {{checkboxModel.virtualbox.minimumVersion}} or newer 5.0.x release. Please uninstall detected Oracle VirtualBox and restart Installer to continue.</span>
     </div>
 
+    <div class="message-container error-message" ng-show="virtualization === false">
+      <span class="pficon pficon-error-circle-o"></span>
+      <span>Please enable hardware virtualization for your platform to improve performance.</span>
+    </div>
+
+    <div class="message-container error-message" ng-show="virtualization === undefined">
+      <span class="pficon pficon-error-circle-o"></span>
+      <span>Make sure hardware virtualization is enabled on your platform if supported to improve performance.</span>
```

Comment:
Verified using Windows 10 Pro host with disabled Virtualization in BIOS. It works as advertised but there are several cons:
1. Only 32-bit VM can be created
2. minishift fails to start with error: "This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory".